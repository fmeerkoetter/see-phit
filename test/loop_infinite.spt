R"*(
<for var=n from=0 to=5 inc=0>
  <div>
    item{{n}}
  </div>
</for>
)*"_html;
