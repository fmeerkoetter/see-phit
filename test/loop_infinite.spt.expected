Error() [with int ROW = 2; int COL = 29; WHAT = spt::Infinite_loop_in_for_tag]':
